dangerfile.js

```diff
@@ -71,3 +73,27 @@ const snapshotsModified = fileChanges.some(filePath => (
 if (componentSourceFilesModified && !snapshotsModified) {
   warn('Package source files (e.g. `package/packageName/src/packageName.js`) were updated, but snapshots weren\'t. Have you checked that the tests still pass?'); // eslint-disable-line max-len
 }
+
+// Ensure shrinkwrap changes are intentional.
+const shrinkwrapUpdated = includes(modifiedFiles, 'changelog.md');
+if (shrinkwrapUpdated) {
+  warn('`npm-shrinkwrap.json` was updated. Ensure that this was intentional.');
+}
+
+// New files should include the Backpack license heading.
+const unlicensedFiles = createdFiles.filter((filePath) => {
+  if (filePath.match(/\.(js|css|scss|sh)/)) {
+    const fileContent = fs.readFileSync(filePath);
+    return !fileContent.includes('Licensed under the Apache License, Version 2.0 (the "License")');
```

Comment:
`license-header.js` exports the license for uses like this. Better to check against this if possible (but I don't know if that is possible)

Reply:
I did think about that, but I have concerns for a couple of reasons:

* I feel like doing `String.includes` on such a large text snippet might be slower than doing it on a small one.
* `dangerfile.js` exports an XML style comment and a block style one. We'd need to introduce logic to `Dangerfile.js` to know which one of these to use in the check based on the file extension. I dunno - feels a bit like overkill to me for a simple check!

What do you reckon?

Reply:
Sure - I'm a fan of keeping it simple 😸